tests/components/idasen_desk/conftest.py

```diff
@@ -0,0 +1,49 @@
+"""IKEA Idasen Desk fixtures."""
+
+from collections.abc import Callable
+from unittest import mock
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def mock_bluetooth(enable_bluetooth):
+    """Auto mock bluetooth."""
+
+
+@pytest.fixture(autouse=False)
+def mock_desk_api():
+    """Set up idasen desk API fixture."""
+    with mock.patch("homeassistant.components.idasen_desk.Desk") as desk_patched:
```

Comment:
I would have had to ask you to change the mocking method before you moved this code into the new lib since it would have violated https://developers.home-assistant.io/docs/development_testing?_highlight=writing#writing-tests-for-integrations but now thats its patching the pypi code its not a problem anymore.

Reply:
I am not sure I understand the issue. Previously I was also not mocking the `Desk` class. I was mocking the `IdasenDesk` class from the `idasen` library. See https://github.com/home-assistant/core/commit/cc38cdca21adfbc2e9a93bcce36e88bee2aced5e#diff-b9376dd924749fc65fb9dab4bf612ff6f20f8ae841179add3be45cebf8daea0dR18

Was that also an issue?

Reply:
What you were doing before was fine 👍 

I missed it with all the force pushes